clair/clair.go

```diff
@@ -77,9 +79,10 @@ func NewClair(url string) Clair {
 
 func newLayer(image *docker.Image, index int) *layer {
 	var parentName string
-	if index > 0 {
-		parentName = image.FsLayers[index-1].BlobSum
+	if index < len(image.FsLayers)-1 {
+		parentName = image.FsLayers[index+1].BlobSum
```

Comment:
We reversed the loop, but layers are still stored in the original order, why we take upper layer as a parent?